### PR TITLE
api/config/samples: add metrodr_drpolicy example

### PR DESCRIPTION
**Pull request by rakeshgm**
Signed-off-by: rakeshgm <rakeshgm@redhat.com>

### CODE REVIEW

File: config/samples/ramendr_v1alpha1_metrodr_drpolicy.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+apiVersion: ramendr.openshift.io/v1alpha1
+kind: DRPolicy
+metadata:
+  name: metrodr-drpolicy-sample
+spec:
+  schedulingInterval: "1h" # hourly
+  replicationClassSelector:
```
Comment (raghavendra-talur) [inline config/samples/ramendr_v1alpha1_metrodr_drpolicy.yaml:7]: Can probably remove the regional dr specific fields.

Comment (rakeshgm) [reply config/samples/ramendr_v1alpha1_metrodr_drpolicy.yaml:6]: i removed `schedulingInterval`

**Review by raghavendra-talur — APPROVED** [review]

**Review by ShyamsundarR — COMMENTED** [review]
We also bundle the samples, would this need to be bundled as well? IOW, this file needs an edit: https://github.com/RamenDR/ramen/blob/main/config/hub/samples/kustomization.yaml

**Comment by rakeshgm** [comment]
> We also bundle the samples, would this need to be bundled as well? IOW, this file needs an edit: https://github.com/RamenDR/ramen/blob/main/config/hub/samples/kustomization.yaml

I see your point. I will add the sample file for only DRpolicy in this patch.

**Comment by rakeshgm** [comment]
> > We also bundle the samples, would this need to be bundled as well? IOW, this file needs an edit: https://github.com/RamenDR/ramen/blob/main/config/hub/samples/kustomization.yaml
> 
> I see your point. I will add the sample file for only DRpolicy in this patch.

I have made the changes in `kustomization.yaml`

**Review by ShyamsundarR — APPROVED** [review]

**Pull request merged**